extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/src/main/java/org/eclipse/smarthome/binding/mqtt/generic/internal/generic/MqttChannelTypeProvider.java

```diff
@@ -55,14 +56,12 @@
         return types.get(channelTypeUID);
     }
 
-    @SuppressWarnings("deprecation")
     @Override
     public @Nullable ChannelGroupType getChannelGroupType(@NonNull ChannelGroupTypeUID channelGroupTypeUID,
             @Nullable Locale locale) {
         return groups.get(channelGroupTypeUID);
     }
 
-    @SuppressWarnings("deprecation")
     @Override
     public @Nullable Collection<@NonNull ChannelGroupType> getChannelGroupTypes(@Nullable Locale locale) {
```

Comment:
I don't understand. I do implement `ChannelGroupTypeProvider` next to `ChannelTypeProvider`. Splitting the `MqttChannelTypeProvider` wouldn't make sense in this particular case, because the class is already super tiny and used as a pure container only.